lib/widgets/row_stars.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+
+class StarsRow extends StatelessWidget {
+  const StarsRow({
+    super.key,
+  });
+
```

Comment:
Thinking a bit ahead of time, I'd make this widget receive the movie rating! And display the stars with some logic.
Or, if you want to address this in a later time, please add a `// TODO` comment!